HACKING.md

```diff
@@ -548,6 +548,15 @@ The nested suites use some environment variables to configure the suite and the
     NESTED_CUSTOM_IMAGE_URL: Download and use an custom image from this url
 
 
+### Tagging spread tests
+
+Tags are used to categorize spread tests independently of the test suite where it is located.
```

Comment:
I think the subclause should be: "where they are located."  as the main clause says "tests"

Reply:
updated